app/controllers/products_controller.rb

```diff
@@ -19,6 +19,10 @@ def create
     end
 end
 
+def show
+    @product= Product.find(params[:id])
```

Comment:
Nice `show` action.

But remember, the Ruby Style Guide: https://github.com/airbnb/ruby suggests that we use 2 spaces instead of 4 for indentation.